server/views/temporary-accommodation/premises/_editable.njk

```diff
@@ -0,0 +1,89 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+
+{% call govukFieldset({
+  legend: {
+    text: "What the property address?",
+    classes: "govuk-fieldset__legend--m"
+  }
+}) %}
+
+{{ govukInput({
+  label: {
```

Comment:
We found ourselves duplicating a lot of code with these govuk components. Id, name, value etc. are almost always the same so Stu recently introduced some wrappers around them: https://github.com/ministryofjustice/hmpps-approved-premises-ui/tree/main/server/views/applications/components/formFields 
Just flagging in case you wanted to copy them across but understand if you'd prefer not to!

Reply:
Cheers for the heads up, think I'll take a proper look with the new work we're starting next sprint